src/MOLFiniteDifference.jl

```diff
@@ -0,0 +1,15 @@
+struct MOLFiniteDifference{G} <: DiffEqBase.AbstractDiscretization
+    dxs
+    time
+    approx_order::Int
+    grid_align::G
+end
+
+# Constructors. If no order is specified, both upwind and centered differences will be 2nd order
+function MOLFiniteDifference(dxs, time=nothing; approx_order = 2, grid_align=CenterAlignedGrid())
```

Comment:
why centered default?

Reply:
It was previously

Reply:
it only defaulted to centered when Neumann IIRC?